montepy/input_parser/cell_parser.py

```diff
@@ -185,31 +185,99 @@ def geometry_factory(self, p):
                 nodes["start_pad"].append(node)
         return syntax_node.GeometryTree("geom parens", nodes, "()", p.geometry_expr)
 
-    # support for fill card weirdness
     @_(
-        'number_sequence "(" number_sequence ")"',
-        'number_sequence "(" padding number_sequence ")"',
-        'number_sequence "(" number_sequence ")" padding',
-        'number_sequence "(" padding number_sequence ")" padding',
-        'number_sequence ":" numerical_phrase',
-        'number_sequence ":" padding numerical_phrase',
-        # support for TRCL syntax
-        '"(" number_sequence ")"',
-        '"(" number_sequence ")" padding',
-        '"(" padding number_sequence ")" padding',
```

Comment:
I'll admit that I don't follow all of the cell parser changes in this pull request. It looks valid at first glance, but I'm really relying on the tests to confirm that it still works.